src/modules/person/persistence/person.repository.ts

```diff
@@ -629,4 +629,26 @@ export class PersonRepository {
 
         return oldFamiliennameLowerCase !== newFamiliennameLowerCase;
     }
+
+    public async getKoPersUserLockList(): Promise<string[]> {
+        const daysAgo: Date = new Date();
+        daysAgo.setDate(daysAgo.getDate() - 56);
+
+        const filters: QBFilterQuery<PersonEntity> = {
+            $and: [
+                { personalnummer: { $eq: null } },
+                {
+                    personenKontexte: {
+                        createdAt: { $lte: daysAgo }, //Check that createdAt is older than 56 days
+                        rolleId: {
+                            merkmale: { merkmal: RollenMerkmal.KOPERS_PFLICHT },
+                        },
+                    },
+                },
```

Comment:
This *might* need to have a `$some` condition, not sure what the default is when filtering across collections.
Could you make sure this works (Maybe even create a integration test for this case), by creating a person with two personenkontexte:
- One that would qualify the person for locking (Has the kopers-pflicht and is expired)
- One that would not lock the person (Has kopers-pflicht but is not expired)

Anyway, `$some` would make the intention more clear

```typescript
{
    personenKontexte: {
        $some: {
            createdAt: { $lte: daysAgo }, //Check that createdAt is older than 56 days
            rolleId: {
                merkmale: { merkmal: RollenMerkmal.KOPERS_PFLICHT },
            },
        },
    },
},
```